noxfile.py

```diff
@@ -146,7 +146,11 @@ def system(session):
     # Install all test dependencies, then install this package into the
     # virtualenv's dist-packages.
     session.install("mock", "pytest", "google-cloud-testutils", "-c", constraints_path)
-    session.install("-e", ".[tqdm]", "-c", constraints_path)
+    if session.python == "3.9":
```

Comment:
So, we only want to test `db-types` under a single Python version, 3.9 in this case? It's not a range restriction?

Does the 3.9 version have any specific meaning, or is it more like "recent, but not the latest" kind of choice?
If the latter, we could probably say `SYSTEM_TEST_PYTHON_VERSIONS[-2]` to avoid hardcoding.

Reply:
I think that was because we needed some code paths where it wasn't present, but #423 adds db-dtypes as a proper dependency.